api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -1031,6 +1078,19 @@ definitions:
       address:
         description: Event caller.
         type: string
+  Addresses_attributes : 
```

Comment:
I'm not sure it make sens to create an object for that as only one endpoint will use it...